clientv3/concurrency/stm.go

```diff
@@ -23,7 +25,7 @@ import (
 type STM interface {
 	// Get returns the value for a key and inserts the key in the txn's read set.
```

Comment:
Maybe comment should be now `// Get returns the value for keys...`?

Reply:
> Get returns the value for only one key to give the stm backend some flexibility in satisfying the request, so that's why I kept it as "a key".

@heyitsanthony sorry to disturb you, I'm using stm currently, and I digged the relevant code a little. The Get() Method confuses me, could you help clarify the logic behind this? tks.

For my understanding, stm.Get("a", "b", "c"), and rset caches them all, then stm.Get("c", "d"), It seems that "d" can not be fetched through txn and be inserted to rset.

Could you help correct me? tks.